cityofaustin/atd-moped
# Add `is_deleted` column for personnel soft deletes

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9251

## Testing
**URL to test:** 
_This PR needs to be tested either locally or on moped-test._

**Steps to test:**
1. Create a new project and go to the **Team tab**
2. Add:
   - One team member with role of **Inspector**
   - One team member with role of **Designer**
   - Yourself with **any other role** (or JD Maccombs if you are testing locally since we are all JD locally 👤 )
3. Go the the project table and you should see the Inspector and Designer in the **Team Members** column of your new project's row
4. Go to the Dashboard view and you should see your new project under the **My Projects** tab
5. Go back to your project's **Team tab** and remove all members from the project
6. In the project table, the Designer and Inspector should now be removed from the **Team Members** columns of your project's row
7. In the Dashboard view's **My Projects** tab, your new project should no longer be listed

---
#### Ship list
